packages/jaeger-ui/src/components/TracePage/index.js

```diff
@@ -346,10 +306,14 @@ export class TracePageImpl extends React.PureComponent<TracePageProps, TracePage
       return <ErrorMessage className="ub-m3" error={trace.error || 'Unknown error'} />;
     }
 
+    // $FlowIgnore because flow believes Set<string> cannot be assigned to Set<string | number>
+    const findMatches: Set<string | number> = traceGraphView
```

Comment:
If `uiFind` is empty, then filtering the vertices or spans is unnecessary.

Reply:
Both of the filter functions that can be called have a default return for falsy `uiFind` arguments.